worlds/pokemon_rb/rules.py

```diff
@@ -34,16 +34,16 @@ def prize_rule(i):
         "Rival's House - Rival's Sister": lambda state: state.has("Oak's Parcel", player),
         "Oak's Lab - Oak's Post-Route-22-Rival Gift": lambda state: state.has("Oak's Parcel", player),
         "Viridian City - Sleepy Guy": lambda state: logic.can_cut(state, player) or logic.can_surf(state, player),
-        "Route 2 Gate - Oak's Aide": lambda state: logic.oaks_aide(state, state.multiworld.oaks_aide_rt_2[player].value + 5, player),
+        "Route 2 Gate - Oak's Aide": lambda state: logic.oaks_aide(state, state.multiworld.worlds[player].options.oaks_aide_rt_2.value + 5, player),
```

Comment:
Why not use the world, instead of state.multiworld.worlds[player]?

Reply:
Changed.